src/sparseml/pytorch/utils/sparsification_info/configs.py

```diff
@@ -141,19 +203,37 @@ def from_module(
 
     def loggable_items(
         self,
+        non_zero_only: bool = False,
+        percentages_only: bool = True,
```

Comment:
what's the point of these filters?

Reply:
By default, we were returning too much information; these filters ignore counts (only include percentages); and ignores layers where sparsity is 0; similarly a `enabled_only` filter has been added for `SparsificationQuantization` to only include info for actually quantized layers

Reply:
we still want to see zero counts because this could flag down bugs

Reply:
Will add that in a separate PR